benchmarking_hallucination_metrics/benchmark_hallucination_metrics.ipynb

```diff
@@ -0,0 +1,4732 @@
+            "text/plain": [
+              "<Figure size 640x480 with 1 Axes>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "calculate_and_plot_auroc(res_dict[dataset_name], dataset_name)"
```

Comment:
Can you add some text here to reiterate that this plot is only of a subset of the data? The previous time it was mentioned was a few cells up so it might've been missed, and the plot looks weird without that in mind

Reply:
Yes let's make it more clear: this is just a subsample run for you to quickly try out.  Comment out / change this specific line of code to run the full benchmark.

The main purpose of this code should be users to be able to reproduce our benchmark